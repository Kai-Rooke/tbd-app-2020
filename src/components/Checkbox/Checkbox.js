import React from 'react';
import './Checkbox.css';

export default class Checkbox extends React.Component{
    render(){
        return (    
        
            <div class="labelposition">    <br></br>
                  <input className="checkbox" type="checkbox" id="checkboxexample" name="checkboxexample" value="Option"/>
                  <div class="checkboxposition">
            <label class="container">First Checkbox</label>
            </div>           
            </div>


            )
        }
    };