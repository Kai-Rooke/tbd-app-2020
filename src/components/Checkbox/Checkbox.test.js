import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom';
import Checkbox from './Checkbox';

//cleans up 
afterEach(cleanup);

it("Renders Checkbox Component", () => {
    const { asFragment } = render(<Checkbox/>);
    expect(asFragment()).toMatchSnapshot();
});