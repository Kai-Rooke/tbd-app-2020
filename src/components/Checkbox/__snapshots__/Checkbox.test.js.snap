// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Checkbox Component 1`] = `
<DocumentFragment>
  <div
    class="labelposition"
  >
        
    <br />
    <input
      class="checkbox"
      id="checkboxexample"
      name="checkboxexample"
      type="checkbox"
      value="Option"
    />
    <div
      class="checkboxposition"
    >
      <label
        class="container"
      >
        First Checkbox
      </label>
    </div>
  </div>
</DocumentFragment>
`;
